build.fsx

```diff
@@ -187,95 +188,68 @@ Target "PublishNuget" (fun _ ->
 // --------------------------------------------------------------------------------------
 // Generate the documentation
 
-
-let fakePath = "packages" </> "build" </> "FAKE" </> "tools" </> "FAKE.exe"
```

Comment:
we no longer need FAKE to invoke FSI as the new FAKE API provides means for invoking FSI directly